mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -409,6 +409,90 @@ trait Completions { this: MetalsGlobal =>
         NoneCompletion
     }
   }
+
+  private def isComment(str: String, pos: Position): Boolean = {
+    object CommentDelimiter extends Enumeration {
+      type CommentDelimiter = Value
+      val Init, End = Value
+    }
+    // check if the position is between /** and */
+    // or if line contains before char's position //
+    val lineIdx = pos.line - 1
```

Comment:
What if we do it differently. We could use scalameta tokenizer to tokenize part of the file that is in the last tree range? 
We would need to get the text, then call `.tokenize` and find the current at the current offset. That might be shorter and faster.

Additionally I would add this check at the end of completions in Keywords after `matchesPosition` :

```
if kw.matchesPosition(
   ...
) && isNotComment
```

Reply:
I tried to do it as you suggested!
I added a couple of tests and it seems to work in all scala versions (tested using ` +cross/testOnly tests.pc.CompletionKeywordSuite` locally).

I force pushed in this branch, removing the previous changes that are available in [Giggiux:old-fix-scaladoc-no-completions ](https://github.com/Giggiux/metals/tree/old-fix-scaladoc-no-completions)